index.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>CV - Pinchuk Vladislav</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style type="text/css">
+       div {
+        max-width: 1024px;
+        text-align: center;
+       } 
+    </style>
+</head>
+<body>
+    <div>
+    <header>
+        <h1>Pinchuk Vladislav Alexandrovich</h1>
+    </header>
+    <hr>
+    <img src="photo.jpg" width="150" alt="Photo Pinchuk Vladislav">
+     
+    <section>
+        <h3>Event Specialists / Event Manager <br>
```

Comment:
`h4` couldn't be a child of `h3` tag. I would like to not use `<br>` tag for a new line. Because of `<h3>` tag has block model we could awoid using `<br>`. The same as for other uses of `<br>` tag in this PR. Please check the [docs](url)

> `<br>` is useful for writing a poem or an address, where the division of lines is significant.


You can do something like

`<h3>`content`</h3>`
`<h4>`content`</h4>`